src/man/sssd.conf.5.xml

```diff
@@ -2461,6 +2461,11 @@ pam_account_locked_message = Account locked, please contact help desk.
                             Default: <quote>id_provider</quote> is used if it
                             is set and can perform session related tasks.
                         </para>
+                        <note>
+                            In order to have this feature working as expected,
+                            SSSD must be running as "root" and not as the "sssd"
```

Comment:
It is just convention that unprivileged user is called *sssd* by default.
But it is configurable.
```
./configure --help | grep -- -user
  --with-sssd-user=<user> User for running SSSD (root)
```

Therefore would be better to mention just unprivileged user.